edx_lint/cmd/amnesty.py

```diff
@@ -58,8 +58,8 @@ def format_pylint_disables(error_names, tag=True):
             disabled=", ".join(sorted(error_names)),
             tag=tag_str,
         )
-    else:
-        return ""
+
+    return ""
```

Comment:
TBH, I don't really like it either. I'll disable the rule, and undo this change.